src/components/organisms/Dashboard.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { Pane, Select, Text } from 'evergreen-ui';
+import AddComponentButton from '../atoms/AddComponentButton';
+import DashboardItem from '../molecules/DashboardItem';
+import { useReactiveVar } from '@apollo/client';
+import { dashboardItemsVar } from '../../cache/cache';
+import DataWrapper from '../molecules/DataWrapper';
+import ThresholdChart from '../atoms/ThresholdChart';
+import { ParentSize } from '@visx/responsive';
+
+const Dashboard: React.FC = () => {
+    //Apollo local state
+    const dashboardItems = useReactiveVar(dashboardItemsVar);
+
+    return (
+        <Pane width="100%" height="100%" display="flex" flexDirection="column" alignItems="center" paddingBottom="20px">
+            <Pane width="80%" height="100px" position="relative">
+                <Select position="absolute" bottom="20px" width="200px">
```

Comment:
Is it possible to avoid using pixels?

`rem`s are generally better if you don't need very specific control over an elements size.

Reply:
Probably!